src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -431,6 +437,115 @@ private OpenApiOperation CreateBaseOperation(string description, List<OpenApiTag
             return operation;
         }
 
+        /// <summary>
+        /// This method creates a list of OpenApiParameter objects for the input parameters of a stored procedure.
+        /// A input parameter will be marked REQUIRED if default value is not available.
+        /// </summary>
+        private static List<OpenApiParameter> CreateStoredProcedureInputParameters(StoredProcedureDefinition spDefinition)
+        {
+            List<OpenApiParameter> parameters = new();
+
+            foreach ((string paramKey, ParameterDefinition parameterDefinition) in spDefinition.Parameters)
+            {
+                parameters.Add(
+                    GetOpenApiQueryParameter(
+                        name: paramKey,
+                        description: "Input parameter for stored procedure parameter",
+                        required: !parameterDefinition.HasConfigDefault,
+                        type: TypeHelper.GetJsonDataTypeFromSystemType(parameterDefinition.SystemType).ToString().ToLower()
+                    )
+                );
+            }
+
+            return parameters;
+        }
+
+        /// <summary>
+        /// Creates a list of OpenAPI parameters for querying tables and views.
+        /// The query parameters include $select, $filter, $orderby, $first, and $after, which allow the user to specify which fields to return,
+        /// filter the results based on a predicate expression, sort the results, and paginate the results.
+        /// </summary>
+        /// <returns>A list of OpenAPI parameters.</returns>
+        private static List<OpenApiParameter> CreateTableAndViewQueryParameters()
+        {
+            List<OpenApiParameter> parameters = new();
+
+            // Add $select query parameter
+            parameters.Add(
+                GetOpenApiQueryParameter(
+                    name: "$select",
+                    description: "A comma separated list of fields to return in the response.",
```

Comment:
if you want to compare against another deployed microsoft api (Microsoft Graph), see this doc for the descriptions they use for query params: https://learn.microsoft.com/en-us/graph/query-parameters?tabs=http#odata-system-query-options

Reply:
I feel like little more descriptive is good, as it's creates less confusion on the usage.